Python/flowgraph.c

```diff
@@ -1554,7 +1554,7 @@ optimize_basic_block(PyObject *const_cache, basicblock *bb, PyObject *consts)
             case KW_NAMES:
                 break;
             case PUSH_NULL:
-                if (nextop == LOAD_GLOBAL && (inst[1].i_opcode & 1) == 0) {
+                if (nextop == LOAD_GLOBAL && (inst[1].i_oparg & 1) == 0) {
```

Comment:
The other cases you ``bb->b_instr[i+1]`` rather than ``inst[1]``. Shall we do that here to be consistent?